Formula/mvtools.rb

```diff
@@ -52,10 +52,11 @@ def caveats
       import vapoursynth as vs
       vs.core.std.LoadPlugin(path="#{lib/shared_library("libmvtools")}")
     EOS
-    python = Formula["vapoursynth"].deps
-                                   .find { |d| d.name.match?(/^python@\d\.\d+$/) }
-                                   .to_formula
-                                   .opt_bin/"python3"
+    python_formula = Formula["vapoursynth"].deps
+                                           .find { |d| d.name.match?(/^python@\d\.\d+$/) }
+                                           .to_formula
+    python_version = python_formula.name.split("@")[1] # e.g. "3.11"
+    python = python_formula.opt_bin/"python#{python_version}"
```

Comment:
Each python formula exposes an unversioned `python` in `opt_libexec`, so that's prob the easiest here:
```suggestion
    python = Formula["vapoursynth"].deps
                                   .find { |d| d.name.match?(/^python@\d\.\d+$/) }
                                   .to_formula
                                   .opt_libexec/"bin/python"
```

Reply:
Good to know, thank you!